src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -304,7 +304,7 @@ protected void printSettings() {
         logger.info("Inflater: " + (usingIntelInflater ? "IntelInflater": "JdkInflater"));
 
         logger.info("GCS max retries/reopens: " + BucketUtils.getCloudStorageConfiguration(NIO_MAX_REOPENS).maxChannelReopens());
-        logger.info("Using google-cloud-java patch c035098b5e62cb4fe9155eff07ce88449a361f5d from https://github.com/droazen/google-cloud-java/tree/dr_all_nio_fixes");
+        logger.info("Using google-cloud-java patch 6d11bef1c81f885c26b2b56c8616b7a705171e4f from https://github.com/droazen/google-cloud-java/tree/dr_all_nio_fixes");
```

Comment:
As @lbergelson pointed in several PRs, this change shouldn't be done manually. I have been thinking about how to get rid of this manual hack, and maybe an idea is to have a resource file (e.g., java property) parsed by the gradle build and adding dependencies that are snapshots/patches. Then, this property file can be loaded in the CLP to log an info/warning about the state of this dependencies. That may also require to use the hash-based version number instead of the date-based for that kind of dependencies.

That's just a random suggestion...

Reply:
This is temporary until we can move to an official gcloud release that incorporates our patches. For now, we need a way for our production users to be sure that they're running with the right patches.

Reply:
Ok...